src/main/scala/com/nec/spark/planning/LibLocation.scala

```diff
@@ -0,0 +1,38 @@
+package com.nec.spark.planning
+
+import java.nio.file.{Files, Path, Paths, StandardOpenOption}
+
+import com.nec.spark.{RequestCompiledLibraryForCode, RequestCompiledLibraryResponse, SparkCycloneExecutorPlugin}
+
+import org.apache.spark.api.plugin.PluginContext
+
+object LibLocation {
+  trait LibLocation {
+    def resolveLocation(): Path
+  }
+
+  case class LocalLibLocation(libraryPath: String) extends LibLocation {
```

Comment:
Just noticed, `LocalLibLocation` is not used?

Reply:
Gonna delete that.